### PR TITLE
Revert default Windows password

### PR DESCRIPTION
**Pull request by pablo-herranz**
Verification Run: http://openqa.suse.de/tests/12275975

### CODE REVIEW

**Comment by DimStar77** [comment]
Do we need a VR or are we confident this is the only thing that broke by the force-push? (force push changed 74 filed after all)

File: lib/windowsbasetest.pm [hunk 1]
```diff
@@ -122,8 +122,7 @@ sub wait_boot_windows {
     reset_consoles;
     assert_screen 'windows-screensaver', 600;
     send_key_until_needlematch 'windows-login', 'esc';
-    # type_password
-    type_string("password");
+    type_password
```
Comment (DimStar77) [inline lib/windowsbasetest.pm:125]: missing semicolon

**Review by DimStar77 — CHANGES_REQUESTED** [review]

File: lib/windowsbasetest.pm (hunk 1 above)
Comment (pablo-herranz) [reply lib/windowsbasetest.pm]: Fixed

**Comment by pablo-herranz** [comment]
Maybe the 74 files changed are because of rebasing? :thinking:

**Review by ricardobranco777 — APPROVED** [review]

**Review by DimStar77 — APPROVED** [review]

File: lib/windowsbasetest.pm [hunk 2]
```diff
@@ -122,8 +122,7 @@ sub wait_boot_windows {
     reset_consoles;
     assert_screen 'windows-screensaver', 600;
     send_key_until_needlematch 'windows-login', 'esc';
-    # type_password
-    type_string("password");
+    type_password;
```
Comment (m-dati) [inline lib/windowsbasetest.pm:125]: LGTM

**Pull request merged**